app/backend/main.py

```diff
@@ -32,7 +32,9 @@
 
 logger = logging.getLogger("uvicorn.error")
 
-FRONTEND_URL = os.getenv("FRONTEND_URL", "http://localhost:8080")
+FRONTEND_URL = os.getenv("FRONTEND_URL")
+CHROMADB_HOST = os.getenv("CHROMADB_HOST", "localhost")
+SERVER_ROOT_PATH = os.getenv("SERVER_ROOT_PATH", "")
```

Comment:
| Header | Development | Production | Explanation |
|--------|-------------|------------|-------------|
| `FRONTEND_URL` | `http://localhost:8080` | None | In development this is needed for CORS. In production frontend and backend are same origin (one at `/` and one at `/api/`). |
| `CHROMADB_HOST` | `chromadb` | `35.188.136.220` | In development this is according to the docker network created by docker compose. In production this is the external IP of the VM on which chromadb is deployed #51. |
| `SERVER_ROOT_PATH` | Empty string | `/app` | In development the backend is at root of localhost port 8001. In production it is served at `/api/` and we need openapi docs to be aware.